modules/arm_plugin/src/transformations/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "openvino_arm_cpu_transformations")
+
+file(GLOB_RECURSE SOURCES_TRANSFORMATIONS
+        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+file(GLOB_RECURSE HEADERS_TRANSFORMATIONS
+        ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+
+add_library(${TARGET_NAME} STATIC ${SOURCES_TRANSFORMATIONS})
+
+ie_arm_neon_optimization_flags(neon_flags)
+set_source_files_properties(${SOURCES_TRANSFORMATIONS} PROPERTIES COMPILE_OPTIONS "${neon_flags}")
+
+
+target_include_directories(${TARGET_NAME} PRIVATE
+        "${HEADERS_TRANSFORMATIONS}"
```

Comment:
`HEADERS_TRANSFORMATIONS` is a list of headers, while we need to provide an include directory.
I suppose we need to have only `CMAKE_CURRENT_SOURCE_DIR` as an include dir

Reply:
@ilya-lavrenov done